stdlib/src/socket.rs

```diff
@@ -80,14 +80,11 @@ mod _socket {
         vm.ctx.exceptions.os_error.clone()
     }
 
-    #[pyattr(once)]
+    #[pyattr(name = "timeout", once)]
```

Comment:
when `name` is missing, it follows the following identifier's name. Because the function name is `timeout`, this is redundant.